### DISCUSSION TITLE
Fullcalendar support standard plugins

### DISCUSSION THREAD

**Discussion by dorel14** [zauberzeug/nicegui#3516] [Ideas / Feature Requests]
Hello 

It would be nice to support fullcalendar basic plugins and languages , i started a project with nicegui and the fullcalendar example and was surprised to see that the timegrid is not available, same thing for the buttons translation if i set them UP in options 

https://fullcalendar.io/docs/plugin-index

**Comment by falkoschindler** [comment]
Sure, that would be nice. The [fullcalendar example](https://github.com/zauberzeug/nicegui/tree/main/examples/fullcalendar) is just to show how to use a library like fullcalendar in a custom NiceGUI element. You can certainly extend it with plugins.

**Reply by dorel14** [reply]
Hi ,  good news ,  i have made a test for translations 

1.  Download the fullcalendar js package [here ](https://github.com/fullcalendar/fullcalendar/releases/download/v6.1.10/fullcalendar-6.1.10.zip)
2. Extract all the files from zip 
3.  Copy the locales-all.global.min.js in the same folder as index.global.min.js
4. Copy the locales folder in the same folder 
5. Add this in  the js script in the full calendar example name 'fullcalendar.js' 

~~~
await loadResource(window.path_prefix + `${this.resource_path}/locales-all.global.min.js`);
~~~
Buttons are translated 
![image](https://github.com/user-attachments/assets/07c35b44-33c7-4449-ad98-35d06a5f51ff)

**Comment by dorel14** [comment]
Finally after a rainy afternoon of work ,  i have a working calendar with plugins activated 

i have created a new fullcalendar example,  , perhaps i can make a pull request from my fork ?

**Reply by dorel14** [reply]
Pull request made

**Reply by dorel14** [reply]
Here is a link to the example 

[https://github.com/dorel14/NiceGui-FullCalendar_more_Options](https://github.com/dorel14/NiceGui-FullCalendar_more_Options)
